packages/tty/tty.0.0.1/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "A library for interacting with teletype and terminal emulators"
+description:
+  "TTY is a library for directly interacting with teletypes and terminal emulators, including escape sequences, colors, and consuming stdin"
+maintainer: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+authors: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+license: "MIT"
+tags: ["terminal" "ansi" "tty" "teletype" "utf8"]
+homepage: "https://github.com/leostera/tty"
+bug-reports: "https://github.com/leostera/tty/issues"
+depends: [
+  "ocaml" {>= "5.1"}
+  "dune" {>= "3.11" & >= "3.11"}
```

Comment:
```suggestion
  "dune" {>= "3.11" & >= "3.11"}
```
If you remove dune from the dependencies in the dune-project files, this will be generated correctly